Sources/Ink/Internal/FormattedText.swift

```diff
@@ -29,6 +29,8 @@ internal struct FormattedText: Readable, HTMLConvertible {
               modifiers: ModifierCollection) -> String {
         return components.reduce(into: "") { string, component in
             switch component {
+            case .linebreak:
+                string.append("<br />")
```

Comment:
I suggest removing the whitespace from this string, to be inline with all of the other HTML that Plot outputs.